script/lint.js

```diff
@@ -217,7 +217,7 @@ const LINTERS = [{
       }
 
       // Read the patch list
-      const patchFileList = fs.readFileSync(dotPatchesPath, 'utf8').trim().split('\n');
+      const patchFileList = fs.readFileSync(dotPatchesPath, 'utf8').trim().split(/\r?\n/);
```

Comment:
[os.EOL](https://nodejs.org/api/os.html#os_os_eol) might be a better fit here, requires an import btw
```suggestion
      const patchFileList = fs.readFileSync(dotPatchesPath, 'utf8').trim().split(os.EOL);
```

Reply:
I think this regex could be a more friendly way because you never know if the user enabled git core.autocrlf

Reply:
@reitowo it is always disabled: https://github.com/electron/build-tools/blob/07ad97f145f9fb82aee17d76eb38669a6c035122/src/utils/git.js#L20-L25

we override it.

Reply:
There still chance that the user somehow edited manually and saved with either line ending.

Reply:
Sure, but they shouldn't do that, and we don't generally write code in this repo to account for what users could have done in edge cases despite the repo mandating otherwise.